pages/pil/exemptions/routers/modules.js

```diff
@@ -74,7 +99,8 @@ module.exports = () => {
               json: {
                 data: {
                   module,
-                  description: req.form.values[`module-${module}-reason`]
+                  description: req.form.values[`module-${module}-reason`],
+                  species: castArray(req.form.values[`module-${module}-species`]).filter(s => s !== '')
```

Comment:
why is this filter here? do we have empty string animals being added somehow?

Reply:
because one can click on 'Add another', not chose species and then submit, in this case there will be one empty string it the array of species